lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.ts

```diff
@@ -43,25 +56,38 @@ export class SearchChipAutocompleteInputComponent implements OnInit, OnDestroy {
     allowOnlyPredefinedValues = true;
 
     @Output()
-    optionsChanged: EventEmitter<string[]> = new EventEmitter();
+    optionsChanged: EventEmitter<AutocompleteOption[]> = new EventEmitter();
```

Comment:
I think you don't need put type at left side. Just put <AutocompleteOption[]> in EventEmitter at right side